netbox/dcim/models/racks.py

```diff
@@ -357,7 +357,7 @@ def get_rack_units(self, user=None, face=DeviceFaceChoices.FACE_FRONT, exclude=N
 
         return [u for u in elevation.values()]
 
-    def get_available_units(self, u_height=1, rack_face=None, exclude=None):
+    def get_available_units(self, u_height=1, rack_face=None, exclude=None, exclude_from_utilization=False):
```

Comment:
The purpose of the `exclude_from_utilization` parameter isn't immediately clear. Is it really needed? If so, we should rename it to something like `ignore_excluded_devices`.